UPGRADE.md

```diff
@@ -2,6 +2,266 @@
+            replace(replace(replace(dyn.fax, '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.street is not NULL,
+          CONCAT(
+            '\"street\":\"',
+            replace(replace(replace(dyn.street, '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.zip is not NULL,
+          CONCAT(
+            '\"zip\":\"',
+            replace(replace(replace(dyn.zip, '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.city is not NULL,
+          CONCAT(
+            '\"city\":\"',
+            replace(replace(replace(dyn.city, '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.state is not NULL,
+          CONCAT(
+            '\"state\":\"',
+            replace(replace(replace(dyn.state, '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.country is not NULL,
+          CONCAT(
+            '\"country\":\"',
+            replace(replace(replace(dyn.country, '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.function is not NULL,
+          CONCAT(
+            '\"function\":\"',
+            replace(replace(replace(dyn.function, '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.company is not NULL,
+          CONCAT(
+            '\"company\":\"',
+            replace(replace(replace(dyn.company, '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.text is not NULL,
+          CONCAT(
+            '\"text\":\"',
+            replace(replace(replace(dyn.text, '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.textarea is not NULL,
+          CONCAT(
+            '\"textarea\":\"',
+            replace(replace(replace(replace(dyn.textarea, '\r\n', '\\n'), '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.date is not NULL,
+          CONCAT(
+            '\"date\":\"',
+            replace(replace(replace(dyn.date, '"', '\\\"'), '/', '\/'), '\\', '\\\\'),
+            '\"'
+          ),
+          NULL
+        ),
+        IF(
+          dyn.attachment is not NULL,
+          CONCAT(
+            '\"attachment\":',
+            replace(replace(replace(dyn.attachment, '"', '\\\"'), '/', '\/'), '\\', '\\\\')
+          ),
+          NULL
+        ),
+        IF(
+          dyn.checkbox is not NULL,
+          CONCAT(
+            '\"checkbox\":',
+            dyn.checkbox
```

Comment:
@Prokyonn  Sorry it looks like checkbox can be an empty string so I think we need todo here a case when